### PR TITLE
Extract section according to paper_references csv using grobid

### PR DESCRIPTION
**Pull request by erick4556**
Extract section (text and images) according to paper_references csv using grobid. The tables are missing and are still in process

### CODE REVIEW

File: Solar/grobid_extraction/pdf_section_extraction/single_pdf/pdf_xml_extraction.py [hunk 1]
```diff
@@ -42,20 +42,11 @@ def process_fulltext_document(service, file):
         os.makedirs(output_folder_xml)
 
     # Save XML file
-    output_path_xml = os.path.join(output_folder_xml, "paper_1.2.xml")
+    output_path_xml = os.path.join(output_folder_xml, "paper_15721500205e95c1bb016594.80962303.xml")
```
Comment (dgarijo) [inline Solar/grobid_extraction/pdf_section_extraction/single_pdf/pdf_xml_extraction.py:45]: I don't like hardcoded paths. Isn't there a CLI to use this?

**Comment by dgarijo** [comment]
@erick4556 how does this affect the [result specification](https://github.com/oeg-upm/solar-qa/blob/main/result_specification.md)?

File: Solar/grobid_extraction/pdf_section_extraction/single_pdf/pdf_xml_extraction.py (hunk 1 above)
Comment (erick4556) [reply Solar/grobid_extraction/pdf_section_extraction/single_pdf/pdf_xml_extraction.py:45]: I don't know but that part just extract section for one paper, i can get rid of the hardcoded paths

**Comment by erick4556** [comment]
> @erick4556 how does this affect the [result specification](https://github.com/oeg-upm/solar-qa/blob/main/result_specification.md)?

It's just to identify the paper according to its No_de_Ref. I think it doesn't affect the result

**Comment by dgarijo** [comment]
Then where is this extraction represented? What specification does it follow?

**Comment by erick4556** [comment]
> Then where is this extraction represented? What specification does it follow?

The extraction is in:  https://github.com/oeg-upm/solar-qa/blob/main/Solar/result/LLama_8B/ground_truth_782_papers.json

**Comment by dgarijo** [comment]
I mean the extraction of images and their text. Where does this go?

**Comment by erick4556** [comment]
> I mean the extraction of images and their text. Where does this go?

Ah ok, so far, i'm testing it but i wanted to upload that part, even though it's not still using it to the result, i mean, it's on test

**Pull request merged**